Day9/day9.js

```diff
@@ -0,0 +1,120 @@
+function oneStepMoveHead(direction, headPosition) {
+  if (direction === "U") {
+    headPosition[0]++;
+  } else if (direction === "D") {
+    headPosition[0]--;
+  } else if (direction === "R") {
+    headPosition[1]++;
+  } else if (direction === "L") {
+    headPosition[1]--;
+  }
+}
+
+function isTwoKnotsNotTouching(oneKnot, nextKnot) {
+  if (
```

Comment:
This can be rewritten as 
```suggestion
    return ! (Math.abs(oneKnot[0] - nextKnot[0]) < 2 && Math.abs(oneKnot[1] - nextKnot[1]) < 2);
```